src/helpers.js

```diff
@@ -3,6 +3,7 @@ const osName = require('os-name');
 const path = require('path');
 const packages = require('./packages');
 const utils = require('./utils');
+const { getNdk } = require('./packages/get-ndk');
```

Comment:
I dont think this needs to be in a separate file. Helpers are meant to be exportable as a single function from a single place for now.

Reply:
I actually write a test case for this function. I am actually thinking moving out all function in `helper` to make every function more testable and the helper function not that crowded.

Reply:
I think this can be our first try.

Reply:
I will put it back tomorrow.